test/testenv/buildx.go

```diff
@@ -261,13 +304,14 @@ func (b *BuildxEnv) RunTest(ctx context.Context, t *testing.T, f TestFunc) {
 		f(ctx, gwc)
 		return gwclient.NewResult(), nil
 	}, ch)
-	if err != nil {
-		t.Fatal(err)
```

Comment:
This fatal call here could sometimes make the test runner yell because a buildkit status message could hit the test log after the fatal is called.
Hence why I moved the `<-done` after.